### PR TITLE
Minor. Update api doc about source regarding json response for email fields. RD-23635

### PR DESCRIPTION
**Pull request by onima**
email_api_swagger

This will be displayed on 4 endpoints for source (2 GET and 2 PUT)

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -8483,6 +8487,78 @@ components:
...
+          description: Use STARTTLS protocol to establish secure SMTP connection. Only on Email sources
+          type: boolean
+        smtp_authentication_type:
+          description: SMTP Authentication type ("basic" or "microsoft_identity_platform"). Only on Email sources
+          type: string
+        smtp_ssl_verify_mode:
+          description: SMTP SSL verification mode ("NONE", "PEER", "CLIENT_ONCE" or "FAIL_IF_NO_PEER_CERT"). Only on Email sources
+          type: string
+        smtp_helo_domain:
+          description: Used to identify HELO/EHLO when connecting to the server. Only on Email sources
```
Comment (onima) [inline specs/engage-digital_openapi3.yaml:8543]: What is the helo domain ? I'm not sure to understand it

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -8483,6 +8487,78 @@ components:
...
+          description: Use STARTTLS protocol to establish secure SMTP connection. Only on Email sources
+          type: boolean
+        smtp_authentication_type:
+          description: SMTP Authentication type ("basic" or "microsoft_identity_platform"). Only on Email sources
+          type: string
+        smtp_ssl_verify_mode:
+          description: SMTP SSL verification mode ("NONE", "PEER", "CLIENT_ONCE" or "FAIL_IF_NO_PEER_CERT"). Only on Email sources
+          type: string
+        smtp_helo_domain:
+          description: Used to identify HELO/EHLO when connecting to the SMTP server. Only on Email sources
```
Comment (onima) [inline specs/engage-digital_openapi3.yaml:8543]: What is the helo domain ? I'm not sure to understand it

Comment (kalilz4485) [reply specs/engage-digital_openapi3.yaml:8543]: absolutely no idea, Yoann should know

**Comment by kalilz4485** [comment]
LGTM

**Pull request merged**